### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by anaghkrckr**

### CODE REVIEW

**Comment by sachinskumar41** [comment]
Add proper PR message and description

**Review by sachinskumar41 — COMMENTED** [review]
Avoid pushing dlls and unwanted empty files

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/bin/*
```
Comment (sachinskumar41) [inline .gitignore:2]: Avoid pushing these files

File: Program.cs [hunk 2]
```diff
@@ -1,18 +1,81 @@
...
+            Program staff=new Program();
+            
+            staff.SelectStaff();  
+            
+          
+        }
+
+
+        public void SelectStaff(){
+            int Options;
```
Comment (sachinskumar41) [inline Program.cs]: Always use camelCase with method arguments and local variables.

File: Program.cs [hunk 3]
```diff
@@ -1,18 +1,81 @@
...
+
         }
+
+        private void MainActions(Staff staff){
+            int Options;
+            bool ContinueMainActions=true;
+            string staffType=staff.GetType().Name.ToUpper();
+            do{
+               
+
```
Comment (sachinskumar41) [inline Program.cs]: Remove unwanted spacing

File: Program.cs [hunk 4]
```diff
@@ -1,18 +1,81 @@
...
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello welcome World!");
-            Staff s;
-            s=new Teacher();
-            s.add();
+            Program staff=new Program();
+            
+            staff.SelectStaff();
```
Comment (sachinskumar41) [inline Program.cs]: Cant the method be static? A single copy is needed here right

File: Program.cs [hunk 5]
```diff
@@ -1,18 +1,81 @@
...
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello welcome World!");
-            Staff s;
-            s=new Teacher();
-            s.add();
+            Program staff=new Program();
+
```
Comment (sachinskumar41) [inline Program.cs]: Object reference invoke can be avoided therefore

File: staffs/administrator.cs [hunk 6]
```diff
@@ -1,11 +1,58 @@
 
 using System;
 namespace training{
-    class Administrator :Staff{
-        public override void add(){
+    public class Administrator :Staff{
+        private string administratorDepartment;
+
```
Comment (sachinskumar41) [inline staffs/administrator.cs]: Unwanted spacing

File: staffs/administrator.cs [hunk 7]
```diff
@@ -1,11 +1,58 @@
...
+        public string AdministratorDepartment{
+            get{return administratorDepartment;}
+            set{administratorDepartment=value;}
```
Comment (sachinskumar41) [inline staffs/administrator.cs]: Since there is no additional logic implemented here, empty setters are fine here

Comment (sachinskumar41) [reply staffs/administrator.cs]: ex: public string AdministratorDepartment{get;set;}, Compiler automatically does what you did here

File: staffs/administrator.cs [hunk 8]
```diff
@@ -1,11 +1,58 @@
...
+
+         public override void ViewStaff(){
+            Console.WriteLine("ADMINISTRATOR");
+            Console.WriteLine("StaffId\tName\tAge\tDepartment");
+            Console.WriteLine(this.StaffId+"\t"+this.StaffName+"\t"+this.StaffAge+"\t"+this.AdministratorDepartment);
+        }
+
+
+        public override void UpdateStaff(){
+            int UpdateOptions;
```
Comment (sachinskumar41) [inline staffs/administrator.cs]: naming conventions

File: staffs/staff.cs [hunk 9]
```diff
@@ -1,12 +1,44 @@
...
+        
 
-    abstract class Staff{
-        public abstract void add();
-        public abstract void delete();
-        public abstract void view();
-        public abstract void update();
+        public int StaffId{
+            get{return staffId;}
+            set{staffId=value;}
```
Comment (sachinskumar41) [inline staffs/staff.cs]: See the previous comment. empty setters works

File: staffs/staff.cs [hunk 10]
```diff
@@ -1,12 +1,44 @@
...
+            get{return staffName;}
+            set{staffName=value;}
+        }
+        public int StaffAge{
+            get{return staffAge;}
+            set{staffAge=value;}
+        }
+
+
+
```
Comment (sachinskumar41) [inline staffs/staff.cs]: Spacing

File: staffs/administrator.cs (hunk 7 above)
Comment (anaghkrckr) [reply staffs/administrator.cs]: didnt get the meaning of empty setters. can you give me an example code.

**Pull request closed without merging**